socorro/unittest/middleware/test_middleware_app.py

```diff
@@ -589,8 +596,17 @@ def put(self, server, url, data=None, expect_errors=False):
 
     def test_overriding_implementation_class(self):
         config_manager = self._setup_config_manager({
-            'implementations.service_overrides': 'Backfill: psql, Bugs: typo'
-        })
+            'implementations.implementation_list': (
+                'psql:socorro.unittest.middleware'
+            ),
+            'implementations.service_overrides': (
+                'Bugs: typo'
+            )
+        },
+            services_list=(
+                ('/bugs/', 'bugs.Bugs'),
+            )
```

Comment:
I'm not too sure what this does, but I suspect it is all for the sake of keeping the Bugs service around, so that you don't have to change the tests that use it? If so, wouldn't it be better to create a fake service that we can use to replace the Bugs one, and make the tests better isolated in the process?